Styles/style.css

```diff
@@ -0,0 +1,54 @@
+@import url(reset.css);
+
+header {
+  height: 80px;
+  width: 100%;
+  background-color: grey;
+}
+
+header h1 {
+  font-size: 30px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 80px;
+}
+
+main {
+  display: inline-block;
+  height: calc(100vh - 200px);
+  width: 80%;
+  background-color: aqua;
+  text-align: center;
+}
+
+main h2 {
+  font-size: 25px;
+  font-weight: 700;
+}
+
+aside {
+  float: right;
+  height: calc(100vh - 200px);
+  width: 20%;
+  background-color: crimson;
+  text-align: center;
+}
+
+aside h2 {
+  font-size: 25px;
+  font-weight: 700;
+}
+
+footer {
+  display: flex;
```

Comment:
don't need to use flex for text-align in footer, it's for more complex blocks, you can align text in footer with text-align and line-height

flex is better for `<main>` and `<aside>` to align in a row, but need to add a container for them